extensions/azurecore/package.json

```diff
@@ -193,8 +193,10 @@
 	"devDependencies": {
 		"@types/mocha": "^5.2.5",
 		"@types/node": "^10.12.12",
-		"@types/request": "^2.48.1",
-		"mocha": "^5.2.0",
+    "@types/request": "^2.48.1",
+    "mocha": "^5.2.0",
```

Comment:
So it looks like we've been lucky so far in that vscode has a dependency on Mocha - so there's extensions like admin-tool-ext that don't actually specify a dev dependency on Mocha but don't break because vscode pulls it in for them. We should probably fix that (also because the mocha version for vscode is really old - 2.3.3)